plugins/catalog-backend-module-gcp/alpha-api-report.md

```diff
@@ -0,0 +1,12 @@
+## API Report File for "@backstage/plugin-catalog-backend-module-gcp"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { BackendFeature } from '@backstage/backend-plugin-api';
+
+// @alpha
+export const catalogModuleGcpGkeEntityProvider: () => BackendFeature;
```

Comment:
I think we can move these exports to public api rather than `alpha` now to be honest.

Reply:
fixed